### PR TITLE
db problem

### PR DESCRIPTION
**Pull request by Z1rael888**
I created a POST,GET,DELETE endpoints for project model, but the issue that,when i create a project in swagger everything is okay
photo_2023-11-19_17-09-13
i got that,evrt is cool, but in sql server management studio i got zeros in Country and Category(which are enums)
photo_2023-11-19_17-09-17
So thats the issue

### CODE REVIEW

File: Api/Routes/CreateProjectEndpoint.cs [hunk 1]
```diff
@@ -17,11 +17,15 @@ public void AddRoutes(IEndpointRouteBuilder app)
                 Title = request.Title,
                 Category = request.Category,
                 Description = request.Description,
-                Country = request.Country
+                Country = request.Country,
+                CreatedOnUtc = request.CreatedOnUtc,
```
Comment (dimanoit) [inline Api/Routes/CreateProjectEndpoint.cs:21]: Good to implement through the interceptor to reduce code duplication 
https://github.com/jasontaylordev/CleanArchitecture/blob/main/src/Infrastructure/Data/Interceptors/AuditableEntityInterceptor.cs
Like here

File: Application/Application.csproj [hunk 2]
```diff
@@ -7,24 +7,24 @@
...
+        <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.8.0" />
+        <PackageReference Include="MediatR" Version="12.1.1" />
+        <PackageReference Include="Microsoft.AspNetCore.Routing" Version="2.2.2" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.13" />
         <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.13">
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.13"/>
+        <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.13" />
```
Comment (dimanoit) [inline Application/Application.csproj:19]: Need to delete followed packages 
Carter, AspNet.Routing, EF.SqlServer, EF.Design

File: Application/Features/Projects/CreateProject.cs [hunk 3]
```diff
@@ -16,6 +16,8 @@ public class Command : IRequest<Result<Guid>> //<Result<Guid>>
         public string Description { get; set; } = string.Empty;
         public Category Category { get; set; }
         public Country Country { get; set; }
+        public DateTime CreatedOnUtc { get; set; }
+        public DateTime? PublishedOnUtc { get; set; }
```
Comment (dimanoit) [inline Application/Features/Projects/CreateProject.cs:20]: What is difference between `CreatedOnUtc` and `PublishedOnUtc`?

File: Domain/Enums/Category.cs [hunk 4]
```diff
@@ -1,5 +1,7 @@
-﻿namespace Domain.Enums;
+﻿using System.Text.Json.Serialization;
 
+namespace Domain.Enums;
+[JsonConverter(typeof(JsonStringEnumConverter))]
```
Comment (dimanoit) [inline Domain/Enums/Category.cs:4]: Shouldn't be here

**Review by dimanoit — APPROVED** [review]

File: Application/Features/Projects/CreateProject.cs (hunk 3 above)
Comment (Z1rael888) [reply Application/Features/Projects/CreateProject.cs:20]: поки шо не придумав))))))))))

**Comment by Z1rael888** [comment]
I dont know what should i put instead of null, i thought maybe AccountId, but it havent worked for me
image
I made RegisterUserCommand like that 
image
I dont know if thats the best idea to handle registration,but one more question is how to make a group for that endpoints, because the problem is that all my endpoints in swagger are in Api group

**Pull request merged**